src/validators/resolve/validateAlias.js

```diff
@@ -0,0 +1,13 @@
+import _ from 'lodash'
+import {friendlyTypeOf} from '../../utils'
+
+export default function validateAlias(alias) {
+  if (!_.isPlainObject(alias)) {
+    return 'Alias should be a plain object'
```

Comment:
No need to say Alias because that will be in the error message. But I'm fine either way.